src/network/Network.php

```diff
@@ -126,22 +127,34 @@ public function unregisterInterface(NetworkInterface $interface) : void{
 	}
 
 	/**
-	 * Sets the server name shown on each interface Query
+	 * Sets the server name and sub name shown on each interface Query
 	 */
-	public function setName(string $name) : void{
+	public function setName(string $name, string $subName = "PocketMine-MP") : void{
 		$this->name = $name;
+		/**
+		 * This is a hack to fix ip dummy address
```

Comment:
I have no idea what this is supposed to mean.

Reply:
when you are "" the motd changes to dummy address like 127.0.0.1 and second is 2001:0db8:85a3:0000:0000:8a2e:0370:7334

Reply:
That's not the point. The point is that the comment is inadequate.